.github/workflows/qns-comment.yml

```diff
@@ -0,0 +1,56 @@
+# Post test results as pull request comment.
+#
+# This is done as a separate workflow as it requires write permissions. The
+# tests itself might run off of a fork, i.e. an untrusted environment and should
+# thus not be granted write permissions.
+
+name: Comment on the pull request
+
+on:
+  workflow_run:
+    workflows: ["QUIC Network Simulator"]
+    types:
+      - completed
+
+jobs:
+  upload:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request'
+    steps:
+      - name: Download comment-data
+        uses: dawidd6/action-download-artifact@v2.11.0
```

Comment:
The `mozilla` org has an allowlist for non-Mozilla actions, so
* dawidd6/action-download-artifact
* thollander/actions-comment-pull-request
* mxinden/quic-interop-runner@action (or its replacement)
would need to be added to that allowlist. Can be requested, but will take some time.

Reply:
I think I need to make the request.

Reply:
For thollander/actions-comment-pull-request, [I made the request](https://bugzilla.mozilla.org/show_bug.cgi?id=1883011).

Reply:
> Can you use https://github.com/actions/download-artifact instead of https://github.com/dawidd6/action-download-artifact? The former is already allowlisted.

Good idea. Switched with the latest commits. See test run on https://github.com/mxinden/quic-interop-runner-action/pull/5. (Also shows the new Quic Interop Runner `--must-include` flag https://github.com/quic-interop/quic-interop-runner/pull/355.)

Reply:
@mxinden `thollander/actions-comment-pull-request` should now be allowlisted.

Reply:
Thank you @larseggert :pray: